### ISSUE TITLE
[suggest] Eye Interact

### ISSUE THREAD

**Issue by Otacon-404** [franfdezmorales/interact#2]
if it is possible to exit from interacting when releasing the ALT key?

and sometimes I have to press twice to start picking from a list

**Comment by franfdezmorales** [comment]
> if it is possible to exit from interacting when releasing the ALT key?
> 
> and sometimes I have to press twice to start picking from a list

I dont understand what do you mean with exit from interact, actually, if u release the ALT key, the eye disappear unless you clicked, UI only disappear with ESC or BACKSPACE, do you mean exit from UI with ALT too?

Im gonna review what could fail detecting the click.

Ty for the suggestion!

**Comment by Otacon-404** [comment]
yeah, I meant after I click on the UI, instead of closing by ESC it should close when I release the ALT key. I think its more practical

**Comment by franfdezmorales** [comment]
> yeah, I meant after I click on the UI, instead of closing by ESC it should close when I release the ALT key. I think its more practical

Hmmm i think is not a good idea because ALT is default key but u can change to another. By the way, you can change the keys on ui/src/App.jsx, the keycode u can find on: https://keycode.info/

**Comment by Otacon-404** [comment]
ok, thank you for your help. there is the issue of not registering my click sometimes. great work tho

**Comment by franfdezmorales** [comment]
> ok, thank you for your help. there is the issue of not registering my click sometimes. great work tho

Must be fixed on [#43ccb0c](https://github.com/franfdezmorales/interact/commit/43ccb0ccfbcd7f2f5449ff905b3827623de3bcb2), waiting for ur reply to close this issue.

Appreciated for the feedback!

**Comment by Otacon-404** [comment]
still having this problem but it seems less than before

**Comment by franfdezmorales** [comment]
Could you send me some repros? Tried like 100 times and works correctly.

**Comment by Otacon-404** [comment]
i will try to record something to show you

**Comment by Otacon-404** [comment]
https://streamable.com/3fmivq
